dev-requirements.txt

```diff
@@ -16,6 +16,7 @@ Django>=1.6.5
 bottle>=0.12.3
 tornado>=4.0
 pyramid>=1.5.2
+webapp2==2.5.2
```

Comment:
Google app engine pins webapp2 to specific versions. I did the same here just in case they release a new version to pypi that doesn't match the available prod environment.